### PR TITLE
docs(ai): add upscale api reference

### PR DESCRIPTION
**Pull request by rickstaa**
This pull request ensures that the AI API reference contains the new upscale pipeline.

@eliteprox to fix your the issue you reported in the Discord server. Please merge this into the main branch after your review. Thanks 🙏🏻!

### CODE REVIEW

**Review by 0xcadams — APPROVED** [review]
LGTM

**Review by eliteprox — APPROVED** [review]
LGTM also

File: ai/orchestrators/models-config.mdx [hunk 1]
```diff
@@ -27,6 +27,11 @@ currently **recommended** models and their respective prices.
     "model_id": "ByteDance/SDXL-Lightning",
     "price_per_unit": 4768371
   },
+  {
+    "pipeline": "upscaling",
```
Comment (eliteprox) [inline ai/orchestrators/models-config.mdx:31]:
```suggestion
    "pipeline": "upscale",
```
The correct name for the pipeline is `upscale`

**Review by eliteprox — CHANGES_REQUESTED** [review]
One edit to the models-config

**Review by eliteprox — APPROVED** [review]

File: ai/orchestrators/models-config.mdx (hunk 1 above)
Comment (rickstaa) [reply ai/orchestrators/models-config.mdx]: Ah thanks for catching that one!

**Pull request merged**